test/functional/apps/discover/_data_grid.ts

```diff
@@ -23,7 +23,8 @@ export default function ({
     const testSubjects = getService('testSubjects');
 
     before(async function () {
-      await esArchiver.load('discover');
+      await kibanaServer.savedObjects.clean({ types: ['search', 'index-pattern'] });
+      await kibanaServer.importExport.load('discover');
```

Comment:
Should there be a new discover.ndjson file in this PR that gets loaded?

Reply:
That was added in https://github.com/elastic/kibana/pull/92526 https://github.com/elastic/kibana/blob/master/test/functional/fixtures/kbn_archiver/discover.json